package.json

```diff
@@ -23,19 +23,23 @@
     "@angular/platform-browser-dynamic": "~8.2.11",
     "@angular/router": "~8.2.11",
     "@avatsaev/three-orbitcontrols-ts": "^0.1.5",
+    "@types/webgl2": "^0.0.5",
     "bootstrap": "^4.3.1",
     "core-js": "^2.6.10",
     "rxjs": "~6.5.3",
     "three": "^0.109.0",
-    "three-obj-mtl-loader": "github:rosulg/three-obj-mtl-loader#b1d0d6183f891984aa30dfe8cbf29217ac4ce133",
+    "three-addons": "^1.2.0",
```

Comment:
Are the following packages needed?
```
    "three-addons": "^1.2.0",
    "three-mtl-loader": "^1.0.2",
    "three-obj-loader": "^1.1.3",
    "three-obj-mtl-loader": "^1.0.3",
```
From the code you have submitted in the PR then I see now reason why they should be added.
If they are not needed then please remove them and keep ``` "three-obj-mtl-loader": "github:rosulg/three-obj-mtl-loader#b1d0d6183f891984aa30dfe8cbf29217ac4ce133"```  as this one works with obj, mtl whilst the original one still has issues with loading the textures.